bpf/sockops/bpf_sockops.h

```diff
@@ -61,7 +61,7 @@ static __always_inline void sk_extract4_key(struct bpf_sock_ops *ops,
 	key->family = ENDPOINT_KEY_IPV4;
 
 	key->sport = (bpf_ntohl(ops->local_port) >> 16);
-	key->dport = ops->remote_port >> 16;
+	key->dport = READ_ONCE(ops->remote_port) >> 16;
```

Comment:
add a comment here about clang 7.0 generating a u16 read that is unsupported on some kernels.

Reply:
7.1 or 8.0, but yeah. I'll fix this up